ci/build.ps1

```diff
@@ -29,6 +29,26 @@ function exitIfFailed() {
   }
 }
 
+# https://github.com/lukesampson/scoop#installation
+$scoop = (New-Object System.Net.WebClient).DownloadString('https://get.scoop.sh')
+& {
+  Set-StrictMode -Off
+  Invoke-Expression $scoop
+}
+
+scoop install diffutils nodejs-lts perl
+diff3 --version
+node --version
+npm.cmd --version
+perl --version
+cpanm.bat --version
+
+cpanm.bat -n Neovim::Ext
+if ($LastExitCode -ne 0) {
+  Get-Content -Path "$env:USERPROFILE\.cpanm\build.log"
+}
+perl -W -e 'use Neovim::Ext; print $Neovim::Ext::VERSION'; exitIfFailed
+
```

Comment:
this new block should be guarded by

    if (-not $NoTests) {

else it defeats use cases like https://github.com/neovim/neovim/pull/11654